awx/ui/client/test/e2e/tests/test-credentials-read-only.js

```diff
@@ -0,0 +1,94 @@
+import uuid from 'uuid';
+
+
+let testID = uuid().substr(0,8);
+
+
+let store = {
+    auditor: {
+        username: `auditor-${testID}`,
+        first_name: 'auditor',
+        last_name: 'last',
+        email: 'null@ansible.com',
+        is_superuser: false,
+        is_system_auditor: true,
+        password: 'password'
+    },
+    adminCredential: {
+        name: `adminCredential-${testID}`,
+        description: `adminCredential-description-${testID}`,
+        inputs: {
+            username: 'username',
+            password: 'password',
+            security_token: 'AAAAAAAAAAAAAAAAAAAAAAAAAA'
+        }
+    },
+    created: {}
+};
+
+
+module.exports = {
+    before: function (client, done) {
+        const credentials = client.page.credentials();
+
+        client.login();
+        client.waitForAngular();
+
+        client.inject([store, '$http'], (store, $http) => {
+
+            let { adminCredential, auditor } = store;
+
+            return $http.get('/api/v2/me')
+                .then(({ data }) => {
+                    let resource = 'Amazon%20Web%20Services+cloud';
+                    adminCredential.user = data.results[0].id;
+
+                    return $http.get(`/api/v2/credential_types/${resource}`);
+                }).then(({ data }) => {
```

Comment:
I prefer `.then` on a new line like you have on line 42. It's not a strong preference, but it's what is already in place in the refactored `lib/` and `features/` code.

Reply:
I have no preference here and so will default to your preference. Done :)